ds3-sdk/src/main/java/com/spectralogic/ds3client/commands/spectrads3/ReplicatePutJobSpectraS3Request.java

```diff
@@ -46,21 +55,55 @@ public ReplicatePutJobSpectraS3Request withConflictResolutionMode(final Replicat
         return this;
     }
 
-    @Override
     public ReplicatePutJobSpectraS3Request withPriority(final Priority priority) {
-        super.withPriority(priority);
+        this.priority = priority;
+        this.updateQueryParam("priority", priority);
         return this;
     }
 
 
+    @Override
+    public HttpVerb getVerb() {
+        return HttpVerb.PUT;
+    }
+
+    @Override
+    public String getPath() {
+        return "/_rest_/bucket/" + this.bucketName;
+    }
+    @Override
+    public InputStream getStream() {
+        if (Guard.isStringNullOrEmpty(requestPayload)) {
+            return null;
+        }
+        final byte[] stringBytes = requestPayload.getBytes();
```

Comment:
Specifying a character set will throw an exception. Should the exception propagate, or should I add a LOG to this request to log the potential error?